test/upgrade/prober/wathola/event/services.go

```diff
@@ -122,7 +144,11 @@ func (f *finishedStore) reportViolations(finished *Finished) {
 			times = 0
 		}
 		if times != 1 {
-			f.errors.throw("event #%v should be received once, but was received %v times",
+			throwMethod := f.errors.throwMissing
+			if times != 0 {
```

Comment:
Nit: I think `if times > 1` will be more intuitive and accurate since it's an `unsigned int` type.